localstack/aws/handlers/service_plugin.py

```diff
@@ -5,6 +5,7 @@
 from localstack.http import Response
 from localstack.services.plugins import Service, ServiceManager
 
+from ...utils.threads import SynchronizedDict
```

Comment:
i prefer the absolute imports for utils to indicate coupling to external modules.

```suggestion
from localstack.utils.threads import SynchronizedDict
```